src/unfccc_ghg_data/unfccc_reader/Cabo_Verde/config_cpv_bur1.py

```diff
@@ -0,0 +1,367 @@
+        "M.LULUCF": {"sources": ["3.B"]},
+        # Only 3.C.7 in table, but values are all zero or empty
+        "M.3.C.AG": {
+            "sources": [
+                "3.C.1",
+                "3.C.2",
+                "3.C.3",
+                "3.C.4",
+                "3.C.5",
+                "3.C.6",
+                "3.C.7",
+                "3.C.8",
+            ]
+        },
+        # 3.D.2 is all zeros
+        "M.3.D.AG": {"sources": ["3.D.2"]},
+        "M.AG.ELV": {
+            "sources": ["M.3.C.AG", "M.3.D.AG"],
+        },
+        "M.AG": {"sources": ["3.A", "M.AG.ELV"]},
+        "M.0.EL": {
+            "sources": ["1", "2", "M.AG", "4"],
+        },
+        "4.D": {"sources": ["4.D.1", "4.D.2"]},  # consistency check
+        "1": {"sources": ["1.A"]},  # consistency check
+        "1.A": {
+            "sources": ["1.A.1", "1.A.2", "1.A.3", "1.A.4", "1.A.5"]
+        },  # consistency check
+        "2": {
+            "sources": ["2.A", "2.B", "2.C", "2.D", "2.E", "2.F", "2.G", "2.H"]
+        },  # consistency check
+        "3": {"sources": ["M.AG", "M.LULUCF"]},  # consistency check
+        # "3.A": {"sources": ["3.A.1", "3.A.2"]}, # consistency check
+        "4": {"sources": ["4.A", "4.B", "4.C", "4.D", "4.E"]},  # consistency check
+    },
+    "basket_copy": {
+        "GWPs_to_add": ["AR4GWP100", "AR5GWP100", "AR6GWP100"],
+        "entities": ["HFCS", "FGASES"],
+        "source_GWP": gwp_to_use,
+    },
+}
+
+country_processing_step2 = {
+    "downscale": {
+        "sectors": {
+            "1_all": {
+                "basket": "1",
+                "basket_contents": ["1.A"],
+                "entities": ["CO2", "CH4", "N2O"],
+                "dim": f'category ({coords_terminologies["category"]})',
+            },
+            # Values for 1995/2000/2005/2010/2015/2019 are only available for CO2 and F-gases (table 6)
+            "2_CO2": {
+                "basket": "2",
+                "basket_contents": ["2.A", "2.B", "2.C", "2.D", "2.H"],
+                "entities": ["CO2"],
+                "dim": f'category ({coords_terminologies["category"]})',
+            },
+            "2_KYOTO": {
+                # error of 3.5% for KYOTOGHG (AR6GWP100)
+                "check_consistency": False,
+                "basket": "2",
+                "basket_contents": [
+                    "2.A",
+                    "2.B",
+                    "2.C",
+                    "2.D",
+                    "2.E",
+                    "2.F",
+                    "2.G",
+                    "2.H",
+                ],
+                "entities": [
+                    "KYOTOGHG (SARGWP100)",
+                    "KYOTOGHG (AR4GWP100)",
+                    "KYOTOGHG (AR5GWP100)",
+                    "KYOTOGHG (AR6GWP100)",
+                ],
+                "dim": f'category ({coords_terminologies["category"]})',
+            },
+            "2_FGASES": {
+                # error of 3.5% for KYOTOGHG (AR6GWP100)
```

Comment:
What I still don't understand: where does this incostistency come from? It works for all GWPs except for AR6. It also works for HFCS. The value for 2019 should 35.13 as well.
<img width="967" alt="Screenshot 2024-08-01 at 14 40 58" src="https://github.com/user-attachments/assets/1af84263-61c8-44c6-8469-5e017b9393be">

Reply:
In general there is (currently) no need to downscale f-gas data to subsectors of category 2. In PRIMAP-hist f-gases are currently only resolved at the level of category 2. But this will change in the future, so it doesn't hurt to do the downscaling already.
Your discrepancy comes from using default GWP conversion factors for both the HFCs and the F-gas basket (see above)